### PR TITLE
Move compat data into browser-compat key (webextensions subtree)

### PR DESCRIPTION
**Pull request by sideshowbarker**
Similar to https://github.com/mdn/content/pull/16737, but just for the `mozilla/add-ons/webextensions` subtree, this change, for these reasons:

- consistency with handling of BCD features in other existing docs
- streamlining out unnecessary repetition of BCD feature names
- in keeping with the principle of making use of frontmatter metadata to store data — BCD feature names - rather than having the BCD feature names in the prose body of documents

...this takes BCD feature names out of the arguments to the `Compat` macro, and moves those BCD feature names into the value of the `browser-compat` frontmatter metadata key.

I separated out these changes to the `webextensions` subtree into a different PR than https://github.com/mdn/content/pull/16737 because I’m not as familiar with the contents of these docs, so it seem prudent to ensure the changes get specific review from Web Extensions reviewers.

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/clipboard/index.md [hunk 1]
```diff
@@ -28,7 +29,9 @@ To use this API you need the `"clipboardWrite"` extension [permission](/en-US/do
 
 ## Browser compatibility
 
-{{Compat("webextensions.api.clipboard")}} {{WebExtExamples("h2")}}
+{{Compat}}
+
+{{WebExtExamples("h2")}}
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/clipboard/index.md:34]: This is something we started doing on an ad hoc basis some time ago, moving the examples above the compatibility table so the examples are more visible. So, as we making a change here:
```suggestion
{{WebExtExamples("h2")}}

{{Compat}}
```

File: files/en-us/mozilla/add-ons/webextensions/api/commands/index.md [hunk 2]
```diff
@@ -35,7 +36,9 @@ Listen for the user executing commands that you have registered using the [`comm
 
 ## Browser compatibility
 
-{{Compat("webextensions.api.commands")}} {{WebExtExamples("h2")}}
+{{Compat}}
+
+{{WebExtExamples("h2")}}
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/commands/index.md:41]:
```suggestion
{{WebExtExamples("h2")}}

{{Compat}}
```

File: files/en-us/mozilla/add-ons/webextensions/api/find/index.md [hunk 3]
```diff
@@ -26,4 +27,6 @@ To use this API you need to have the "find" [permission](/en-US/docs/Mozilla/Add
 
 ## Browser compatibility
 
-{{Compat("webextensions.api.find", 1, 1)}} {{WebExtExamples("h2")}}
+{{Compat}}
+
+{{WebExtExamples("h2")}}
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/find/index.md:32]:
```suggestion
{{WebExtExamples("h2")}}

{{Compat}}
```

File: files/en-us/mozilla/add-ons/webextensions/api/menus/index.md [hunk 4]
```diff
@@ -153,7 +154,7 @@ browser.menus.create({
 
 ## Browser compatibility
 
-{{ Compat("webextensions.api.menus", 1, "true") }}
+{{Compat}}
 
 {{WebExtExamples("h2")}}
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/menus/index.md:159]:
```suggestion
{{WebExtExamples("h2")}}

{{Compat}}
```

File: files/en-us/mozilla/add-ons/webextensions/api/pkcs11/index.md [hunk 5]
```diff
@@ -63,4 +64,6 @@ For details about the manifest file's contents and location, see [Native manifes
 
 ## Browser compatibility
 
-{{Compat("webextensions.api.pkcs11", 1, 1)}} {{WebExtExamples("h2")}}
+{{Compat}}
+
+{{WebExtExamples("h2")}}
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/pkcs11/index.md:69]:
```suggestion
{{WebExtExamples("h2")}}

{{Compat}}
```

File: files/en-us/mozilla/add-ons/webextensions/api/search/index.md [hunk 6]
```diff
@@ -25,4 +26,6 @@ To use this API you need to have the `"search"` [permission](/en-US/docs/Mozilla
 
 ## Browser compatibility
 
-{{Compat("webextensions.api.search", 1, 1)}} {{WebExtExamples("h2")}}
+{{Compat}}
+
+{{WebExtExamples("h2")}}
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/search/index.md:31]:
```suggestion
{{WebExtExamples("h2")}}

{{Compat}}
```

File: files/en-us/mozilla/add-ons/webextensions/user_interface/browser_styles/index.md [hunk 7]
```diff
@@ -109,7 +110,7 @@ Most styles are automatically applied, but some elements require you to add the
 
 ## Browser compatibility
 
-{{Compat("webextensions.browser_style")}}
+{{Compat}}
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/user_interface/browser_styles/index.md:113]: `browser_style` refers to a property which is part of the [action](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action), [browser_action](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action), [page_action](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action), [sidebar_action](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/sidebar_action), and [options_ui](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui)) manifest keys. It has no compatibility data of its own.
Perhaps we should replace the macro with:
```suggestion
For browser compatibility information refer to the browser compatibility sections of the [action](en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action#browser_compatibility), ([browser_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action#browser_compatibility), [page_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action#browser_compatibility), [sidebar_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/sidebar_action#browser_compatibility), and [options_ui](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui#browser_compatibility) manifest keys.
```

File: files/en-us/mozilla/add-ons/webextensions/user_interface/browser_styles/index.md [hunk 8]
```diff
@@ -8,6 +8,7 @@ tags:
   - Extensions
   - Guide
   - WebExtensions
+browser-compat: webextensions.browser_style
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/user_interface/browser_styles/index.md:11]:
```suggestion
```

**Review by rebloor — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/add-ons/webextensions/user_interface/browser_styles/index.md [hunk 9]
```diff
@@ -109,7 +109,7 @@ Most styles are automatically applied, but some elements require you to add the
 
 ## Browser compatibility
 
-{{Compat("webextensions.browser_style")}}
+For browser compatibility information refer to the browser compatibility sections of the [action](en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action#browser_compatibility), ([browser_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action#browser_compatibility), [page_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action#browser_compatibility), [sidebar_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/sidebar_action#browser_compatibility), and [options_ui](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui#browser_compatibility) manifest keys.
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/user_interface/browser_styles/index.md:112]:
```suggestion
For browser compatibility information refer to the browser compatibility sections of the [action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action#browser_compatibility), [browser_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action#browser_compatibility), [page_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action#browser_compatibility), [sidebar_action](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/sidebar_action#browser_compatibility), and [options_ui](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui#browser_compatibility) manifest keys.
```

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
@sideshowbarker sorry for the delay, I have approved unmerged. Thanks!

**Comment by sideshowbarker** [comment]
> @sideshowbarker sorry for the delay, I have approved and merged. Thanks!

Cheers — thanks!

**Pull request merged**